subcommands/keys/ca_revoke_device_ca.go

```diff
@@ -179,17 +179,17 @@ func prettyPrintCrl(crlPem string) {
 	}
 	c, err := x509Lib.ParseRevocationList(block.Bytes)
 	subcommands.DieNotNil(err, "Failed to parse generated CRL:")
-	fmt.Println("Certificate Revocation List:")
+	fmt.Println("Certificate revocation list:")
 	fmt.Println("\tIssuer:", c.Issuer)
 	fmt.Println("\tValidity:")
-	fmt.Println("\t\tNot Before:", c.ThisUpdate)
-	fmt.Println("\t\tNot After:", c.NextUpdate)
-	fmt.Println("\tSignature Algorithm:", c.SignatureAlgorithm)
+	fmt.Println("\t\tNot before:", c.ThisUpdate)
+	fmt.Println("\t\tNot after:", c.NextUpdate)
+	fmt.Println("\tSignature algorithm:", c.SignatureAlgorithm)
 	fmt.Println("\tSignature:", hex.EncodeToString(c.Signature))
-	fmt.Println("\tRevoked Certificates:")
+	fmt.Println("\tRevoked certificates:")
 	for _, crt := range c.RevokedCertificates {
 		fmt.Println("\t\tSerial:", crt.SerialNumber.Text(10))
-		fmt.Println("\t\t\tRevocation Date:", crt.RevocationTime)
+		fmt.Println("\t\t\tRevocation date:", crt.RevocationTime)
```

Comment:
These lines 182-192 are done to match the output of the `openssl x509 -text -noout` output so its easy for a user to compare with. They capitalize all this. e.g.
```
openssl x509 -in ./client.pem -text -noout
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            bf:c9:76:32:d4:93:4f:7e:95:b0:ec:32:ef:19:08:93
        Signature Algorithm: ecdsa-with-SHA256
        Issuer: CN = 4b0182b0-9199-4b60-9bf4-70efff1180d8.ota-lite.foundries.io, OU = andy-corp
        Validity
            Not Before: May  8 14:59:23 2023 GMT
            Not After : May  3 14:59:23 2043 GMT
        Subject: CN = bfc97632-d493-4f7e-95b0-ec32ef190893, OU = andy-corp, businessCategory = production
        Subject Public Key Info:
            Public Key Algorithm: id-ecPublicKey
                Public-Key: (256 bit)
                pub:
                    04:b4:e2:3e:2d:6d:93:52:6a:de:87:d3:0e:92:9c:
                    4f:0d:3e:62:b2:bd:aa:ff:4e:9a:6d:f8:94:72:34:
                    2d:7d:99:65:1a:ef:47:d4:a3:99:b0:d9:f3:b6:16:
                    94:98:77:1f:e3:19:a6:59:84:31:2e:96:40:40:02:
                    9c:2c:ac:03:72
                ASN1 OID: prime256v1
                NIST CURVE: P-256
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature
            X509v3 Extended Key Usage: critical
                TLS Web Client Authentication
    Signature Algorithm: ecdsa-with-SHA256
    ```